pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -861,6 +876,19 @@ func loadBalancerServiceIsProgressing(ic *operatorv1.IngressController, service
 		}
 	}
 
+	if platform.Type == configv1.OpenStackPlatformType {
+		wantFloatingIP := getOpenStackFloatingIPInSpec(ic)
+		haveFloatingIP := getOpenStackFloatingIPInStatus(ic)
+		// OpenStack CCM does not support updating Service.Spec.LoadBalancerIP after creation, the load balancer will never be updated.
+		if wantFloatingIP != haveFloatingIP {
+			// TODO(emilien) rewrite this once PR for OCPBUGS-16728 has been merged with a function that standardizes the revert message.
```

Comment:
*nit* I think it's okay to drop this. https://github.com/openshift/cluster-ingress-operator/pull/1142 is still under review and may change, so this more of a prediction of the future rather than a concrete TODO, and might end up going stale.